ublox/Cargo.toml

```diff
@@ -9,7 +9,7 @@ repository = "https://github.com/lkolbly/ublox"
 readme = "../README.md"
 
 [features]
-default = ["std"]
+default = ["std", "serde"]
```

Comment:
Should we disable serde by default?

Reply:
I think serde by default should be fine.